docker-compose.yml

```diff
@@ -35,11 +32,8 @@ services:
 
   app-opensearch:
     container_name: stac-fastapi-os
-    image: stac-utils/stac-fastapi-os
```

Comment:
Can you revert this to pulling from the local code like it was doing before (`stac-utils/stac-fastapi-os`)? The docker-compose file is used for local development, so we want it to build an image from the local code.